tests/kani/Vectors/fixme_702.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+// Failing example from https://github.com/model-checking/kani/issues/702
```

Comment:
Any chance you can add a little description of what the expected behavior of the test is? It's great to have the link here for more details, but this test only has generic names and main function. So you are basically forced to open the link.

Reply:
done